docs/CONFIG.md

```diff
@@ -38,8 +38,9 @@ Configuration lives in [config.yml](../config.yml).
   - `max_concurrent_access_control_syncs`: The maximum number of concurrent access control syncs. Defaults to 1.
   - `job_cleanup_interval`: The interval (in seconds) to run job cleanup task.
   - `log_level`: Connector service log level. Defaults to `INFO`.
-- `connector_id`: The ID of the connector.
-- `service_type` The service type of the connector.
+- `connector_id`: (Deprecated. Configure the connector client in `clients`) The ID of the connector.
+- `service_type`: (Deprecated. Configure the connector client in `clients`) The service type of the connector.
+- `connectors`: A mapping/dictionary between connector id and per-connector configuration, e.g. service type, api key, etc.
```

Comment:
Request: `connectors` has some child fields now. I think it would be nice to document them here as well.